examples/md-flexible/tests/domainDecomposition/MixedBoundaryConditionTest.cpp

```diff
@@ -47,9 +65,10 @@ auto MixedBoundaryConditionTest::setUpExpectations(
           break;
         case ::options::BoundaryTypeOption::reflective:
           // if near a reflective boundary and flying towards it the velocity sign is flipped
```

Comment:
Comment is no longer true right?
```suggestion
          // if near a reflective boundary and flying towards it we expect a repulsive force from reflection
```

Reply:
Yes. Plus the "flying towards it" part is no longer needed.

Reply:
[2d81b4e](https://github.com/AutoPas/AutoPas/pull/724/commits/2d81b4eba1f70d2c841e27e450b169122a5a65bc)